renovate.json

```diff
@@ -48,6 +48,12 @@
 		{
 			"extends": "monorepo:storybook",
 			"prPriority": 1
+		},
+		{
+			"paths": [ "apps/full-site-editing/package.json" ],
+			"packageName": "newspack-blocks",
+			"reviewers": [ "team:@Automattic/create" ],
```

Comment:
I think we should add Grow focus here since they'll handle everything blocks-related. @obenland @creativecoder

Reply:
done. since grow isn't a team, i added Ajax